src/components/atoms/CyToggle.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <label class="cy-toggle">
+    <input type="checkbox" />
+    <span class="slider round"> </span>
+  </label>
+</template>
+<script>
+export default {
+  name: 'CyToggle',
+}
+</script>
+<style scoped>
```

Comment:
please update to `<style lang="sass" scoped>` 
and use the "indented sass style"

This means just deleting all `{` `;` and `}`.